forward/fwd.go

```diff
@@ -138,7 +196,7 @@ func (f *Forwarder) copyRequest(req *http.Request, u *url.URL) *http.Request {
 	// raw query is already included in RequestURI, so ignore it to avoid dupes
 	outReq.URL.RawQuery = ""
 	// Do not pass client Host header unless optsetter PassHostHeader is set.
-	if f.passHost != true {
+	if f.passHost {
```

Comment:
I think actually the previous version was correct - if f.passHost is true, we do not want to modify the original Host header, so it would be exactly:

```go 
if !f.passHost {
     ...
}
```